locale/pt-br/texts.po

```diff
@@ -1754,8 +1755,9 @@ msgstr "Saldo insuficiente"
 msgid ""
 "Ensure your wallet balance covers the required amount to accept this "
 "transaction."
-msgstr "Certifique-se de que o saldo da sua carteira cubra o valor necessário"
-"para aceitar esta transação."
+msgstr ""
+"Certifique-se de que o saldo da sua carteira cubra o valor necessário para "
+"aceitar esta transação."
```

Comment:
This was the only manual change needed for the new and stricter `check_i18n` validation to pass.